closed/OpenJ9.gmk

```diff
@@ -171,6 +171,12 @@ openj9_add_jdk_special = \
 	$(foreach target, $(JDK_OUTPUTDIR)/$(OPENJ9_BIN_OR_LIB_DIR) $(call FindLibDirForModule, java.base), \
 		$(eval $(call openj9_add_jdk_rules, $(target)/$(strip $1), $(OPENJ9_VM_BUILD_DIR)/$(strip $2))))
 
+# jitserver
+
+ifeq (true,$(OPENJ9_ENABLE_JITSERVER))
+$(call openj9_add_jdk_basic, $(JDK_OUTPUTDIR)/bin $(call FindExecutableDirForModule, java.base), jitserver)
```

Comment:
This should say `jitserver$(EXE_SUFFIX)` instead of just `jitserver`.
A copy should also be placed in `$(call FindExecutableDirForModule, java.base)`.

Reply:
Fixed.